elements/pfe-tabs/src/pfe-tabs.scss

```diff
@@ -142,3 +135,38 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
 }
 
 @include pfe-theme-contexts; // imports on="light" etc support
+
+
+/// ===========================================================================
+/// INSET NOTCH
+/// ===========================================================================
+
+@media screen and (min-width: #{pfe-breakpoint(md)}) {
+  :host([vertical][pfe-variant="wind"]) .tabs-prefix {
```

Comment:
Code on line 145-153 is the same as line 154-162; can we combine the selectors here for clarity?

Reply:
i see different selectors... one is prefix the other is suffix.

Reply:
Right but they can be listed together like:
```
:host([vertical][pfe-variant="wind"]) .tabs-prefix,
:host([vertical][pfe-variant="wind"]) .tabs-suffix {
```

since the properties are the same.

Reply:
aha! makes total sense. updating